receiver/awscontainerinsightreceiver/README.md

```diff
@@ -97,7 +97,7 @@ rules:
     resources: ["pods", "nodes", "endpoints"]
     verbs: ["list", "watch"]
   - apiGroups: ["apps"]
-    resources: ["replicasets"]
+    resources: ["replicasets", "daemonsets", "deployments"]
```

Comment:
Will we have to change this in our sample YAMLs and documentation? I think we talked about this previously but I've forgotten by now

Reply:
Yeah good question, It depends on if we change our samples to enable enhanced ci by default or not.  I have a question outstanding for the enhanced ci team on that.  

Right now enhanced ci is not enabled by default so the samples don't need to change.